packages/solana/lib/src/rpc/extension.dart

```diff
@@ -1,6 +1,6 @@
 part of 'client.dart';
 
-extension Convenience on RpcClient {
+extension RpcClientConvenienceMethods on RpcClient {
```

Comment:
Let's just probably call it `RpcClientExt` – it seems to be a pretty common pattern.

Reply:
Yes, of course!